plugins/adr-backend/src/search/DefaultAdrCollatorFactory.ts

```diff
@@ -190,7 +190,10 @@ export class DefaultAdrCollatorFactory implements DocumentCollatorFactory {
       } catch (error: any) {
         // Ignore error if we're able to use cached response
         if (!cacheItem || error.name !== NotModifiedError.name) {
-          throw error;
+          this.logger.error(
+            `Unable to fetch ADRs from ${adrsUrl}: ${error.message}`,
```

Comment:
Could you use stringifyError instead of just the message?

Reply:
ooh neat, was not aware of this utility, updated!